### DISCUSSION TITLE
Trouble with large files?

### DISCUSSION THREAD

**Discussion by jtritter** [MiczFlor/RPi-Jukebox-RFID#2221] [Q&A]
Hi there,

I use RPi-Jukebox for a very long time for my kids and everything works just fine (using Relese 2.3). But now I have problems. I have a large mp3-file (about 110 MB) which works propperly on my PC and phone, but the RPi-Jukebox doesn't like this file. In whe WebInterface the file is shown, can be started, the cover is displayed and the play button appears. But if you click it nothing happens :-( Same with the RFID-Cards :-(
Because this is really the largest file on the box I wonder, if there is a limit of file size. If so, whre can this be changed? What about Release 3, could it be that this large files can be play with this release?
Thanks for help

Yours
Jürgen

**Comment by AlvinSchiller** [comment]
Hi, i recently played files with ~90mb successfully on 2.5.0.
I can try with larger files later.

Can you maybe provide your logs ?
`tail -n 200 /var/log/syslog` 

And which edition you use? Classic or Spotify?

**Comment by s-martin** [comment]
You could also try `mpc listall` to check the content of the mpd database

**Comment by jtritter** [comment]
Thank you for your help. mpc was the trick. The content was not in the database. But an update didn't work as well. So I started playing the files manually with mpc and it worked. Then anonther mpc update added the content to the database. I had to do this with every file larger than 100 MB. Really strange - but it worked.
